src/Microsoft.DocAsCode.Build.Engine/XRefMaps/XRefMapDownloader.cs

```diff
@@ -62,6 +65,25 @@ public async Task<IXRefContainer> DownloadAsync(Uri uri)
             });
         }
 
+        private IXRefContainer ReadLocalFileWithFallback(Uri uri)
+        {
+            var localFilePath = Path.Combine(_baseFolder, uri.OriginalString);
+            if (File.Exists(localFilePath))
+            {
+                return ReadLocalFile(localFilePath);
+            }
+
+            foreach (var fallbackFolder in _fallbackFolders)
+            {
+                var fallbackFilePath = Path.Combine(fallbackFolder, uri.OriginalString);
```

Comment:
The logic inside `foreach` is same as `_baseFolder`. What about always put the `_baseFolder` as the first item in `_fallbackFolders`.

Reply:
I merged `_baseFolder` and `_fallbackFolders` to new list in constructor of XRefMapDonwloader